pysparkling/sql/expressions/mappers.py

```diff
@@ -75,6 +87,115 @@ def __str__(self):
         return "({0} IS NOT NULL)".format(self.column)
 
 
+class Cast(Expression):
+    def __init__(self, e, dtype):
+        super().__init__(e, dtype)
+
+        self.e = e
+        if not isinstance(dtype, str):
+            try:
+                dtype = dtype.typeName()
+            except AttributeError:
+                pass
+        self.dtype = dtype
+        self.dtype_str = dtype
+        if self.dtype_str == 'string':
+            self.dtype = str
+        elif self.dtype_str == 'integer':
+            self.dtype = int
+        elif self.dtype_str == 'boolean':
+            self.dtype = bool
+        elif self.dtype_str in ['str', 'int', 'float', 'bool']:
+            self.dtype = eval(self.dtype_str)
+        else:
+            raise NotImplementedError("Unknown cast type: {}".format(dtype))
+
+    def eval(self, row, schema):
+        return self.dtype(self.e.eval(row, schema))
+
+    def __str__(self):
+        return "cast({0})".format(self.dtype_str)
```

Comment:
I'd prefer to keep one of the name as used by spark, for instance the old column string representation as used by df.show():
```
>>> df.select(df.age.cast('float')).show()
+---+
|age|
+---+
|2.0|
|5.0|
+---+
```

Reply:
Yeah that's good. I'll fix this.